pageserver/src/tenant.rs

```diff
@@ -2695,7 +2667,7 @@ impl Tenant {
         src_timeline: &Arc<Timeline>,
         dst_id: TimelineId,
         start_lsn: Option<Lsn>,
-        ctx: &RequestContext,
+        _ctx: &RequestContext,
```

Comment:
```suggestion
```

Reply:
I'll leave it in there. It's happened often enough to me that I need the `ctx` and was happy to find it as an unused `_ctx` right in my signature. And if we were diligent about removing the ctx every time it become unused for a few weeks until we need it again, then there'd be more needless churn in the code base.